### PR TITLE
fix: response interceptors doesn't work

### PR DESCRIPTION
**Pull request by gitBingxu**
Chain doesn't include the response interceptors

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,12 +5,13 @@
   "main": "index.js",
   "scripts": {
     "test": "grunt test",
-    "start": "node ./sandbox/server.js",
+    "start": "nodemon ./sandbox/server.js",
```
Comment (prabodhmeshram) [inline package.json:8]: Why there are non-related script changes here?

File: package.json [hunk 2]
```diff
@@ -5,12 +5,13 @@
...
     "build": "NODE_ENV=production grunt build",
+    "dev": "NODE_ENV=development grunt build",
     "preversion": "npm test",
     "version": "npm run build && grunt version && git add -A dist && git add CHANGELOG.md bower.json package.json",
     "postversion": "git push && git push --tags",
-    "examples": "node ./examples/server.js",
+    "examples": "nodemon ./examples/server.js",
```
Comment (prabodhmeshram) [inline package.json:14]: Same as above

File: package.json [hunk 3]
```diff
@@ -73,7 +74,8 @@
   "unpkg": "dist/axios.min.js",
   "typings": "./index.d.ts",
   "dependencies": {
-    "follow-redirects": "^1.14.0"
+    "follow-redirects": "^1.14.0",
+    "nodemon": "^2.0.12"
```
Comment (prabodhmeshram) [inline package.json:78]: Do we need this package to support the fix?

File: package.json (hunk 1 above)
Comment (gitBingxu) [reply package.json:8]: sorry,  I pushed some useless code, I will handle it.

File: package.json (hunk 3 above)
Comment (gitBingxu) [reply package.json]: no, we don't need it, I have deleted it.

**Comment by chinesedfan** [comment]
Looks good. If you can add some tests, that will be great.

**Comment by gitBingxu** [comment]
@chinesedfan  Thanks for your advice, \(^o^)/~

**Comment by chinesedfan** [comment]
Close in favor of #4013 with tests.

**Pull request closed without merging**